test/basic.jl

```diff
@@ -100,6 +100,18 @@ end
         @test isfile(hist_file)
         @test isfile(env_file)
 
+
+        # Test that debug prompt generation works
+        @testset "Check debug prompt logic" begin
```

Comment:
I don't think this test has anything to do with "environment and history saving", so this might just be its own testset?

Reply:
Good point; simplified it as well.